File: fax/instance-get-example/instance-get-example.5.x.rb
require 'net/http'
require 'json'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

uri = URI('https://fax.twilio.com/v1/Faxes/FXaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')

Net::HTTP.start(uri.host, uri.port,
                use_ssl: uri.scheme == 'https') do |http|

  request = Net::HTTP::Get.new uri.request_uri
  request.basic_auth(account_sid, auth_token)

  response = http.request request

  puts JSON.parse(response.body)['media_url']
end

File: fax/instance-get-example/instance-get-example.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Fax.V1;
using Twilio.Types;
using System.Collections.Generic;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        const string faxSid = "FXaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";

        TwilioClient.Init(accountSid, authToken);

        var fax = FaxResource.Fetch(faxSid);

        Console.WriteLine(fax.Sid);
    }
}
